### DISCUSSION TITLE
Add native support for MIDS level

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5929] [Feature Requests]
This proposed framework will integrate to clarify the depth of digitization and the minimum information captured at each level:
https://www.tdwg.org/community/cd/mids/

@acbentley mentioned this to me today and it seems like a great suggestion. Elspeth at RBGE is one of the conveners, so it would be a great opportunity to discuss and collaborate with their team if we choose to pursue implementation.

We may be able to automatically calculate the MIDS level for an individual record. We could add these to the form, statistics page, etc.

Minimal implementation would simply to have a `midsLevel` field and allow users to enter a value.

> ## 4.1 Information elements expected at MIDS level 0
> 
> The information elements expected to be present in digital specimen data published at MIDS level 0 are listed in Table 2 below. The terms being developed for the information elements can be found in the TDWG MIDS GitHub site under [MIDS-0](https://github.com/tdwg/mids/issues?q=is%3Aissue+is%3Aopen+label%3AMIDS-0).
> 
> **Table 2: MIDS Level 0: Expected elements, mappings and recommendations for a bare record**
> 
> <table>
>   <tr><th></th>
>    <th>MIDS information element
>    </th>
>    <th>Definition
>    </th>
>    <th>Purpose
>    </th>
>    <th>Required (Bio/Geo/Paelaeo)
>    </th>
>   </tr>
> 
>   <tr><td>1</td>
>    <td> PhysicalSpecimenID <br>
>    </td>
>    <td>A unique identity for the specimen within the curating institution. Whatever the institution uses to uniquely identify the item. For example: DOI, stable identifier, catalogue number, barcode, etc.
>    </td>
>    <td>To allow the curator/researcher to identify the physical specimen the data refer to. To allow citation of the specimen. To enable the attachment of additional information to the specimen record.
>    </td>
>    <td>Yes (all)
>    </td>
>   </tr>
> 
>   <tr><td>2</td>
>    <td>Organization<br>
>   </td>
>    <td>A term to indicate in which institution the specimen is held. This may include an institution code and an institution identifier.
>    </td>
>    <td>To allow a user to locate the physical location of the specimen.
>    </td>
>    <td>Yes (all)
>    </td>
>   </tr>
> 
> </table>

## Example

Here's an example of what fields would be required to meet each level (each level is assumed to have all the values from the level before it)

_Issues arise when you consider users store these values in a number of different fields._ Some of the required information doesn't have a specific field to capture it in Specify, meaning users are likely capturing it in a text field. It's hard to make a generalization in those cases and difficult to auto-calculate.

### MIDS Level 0
* Catalog Number

### MIDS Level 1
* Organization Name (where it physically belongs)
* Determination with a name
* Specimen Type
* Prep Type
* License
* Timestamp Modified 

### MIDS Level 2
* Collecting Event, date, collectors, etc.
* Geological Age (if geo)
* Type
* Attachments/media

and so on and so forth.

Here's the working draft on the current repository:
https://github.com/tdwg/mids/blob/working-draft/current-draft%20/MIDS-definition-v0.17-13Jul2023.md#41-information-elements-expected-at-mids-level-0

**Comment by acbentley** [comment]
This relates to the DiSCCo Collections Digitization Dashboard idea highlighted in this paper - https://public.pensoft.net/items/?p=7TVeXpoqfNYT89tyrm3ifrTeG9Wv8P676JSQp%2FH2pj9hhtoybol4GF7LEbj3fxHT5Fo8esHssdgAZJBvYhfYcmu8YZQoKskWV9Nw4C3JHAyrMHkqqVs%3D&n=kBNhddQ8acsRsM1Z%2FiizL%2BmeUMi4uQ%3D%3D

    A  :             A  :             A  :
 }<(((_°>.,.,.,.}<(((_°>.,.,.,.}<)))_°>
    V                V                V
Andy Bentley
Ichthyology Collection Manager
University of Kansas
Biodiversity Institute
Dyche Hall
1345 Jayhawk Boulevard<x-apple-data-detectors://9/>
Lawrence, KS, 66045-7561<x-apple-data-detectors://9/>
USA<x-apple-data-detectors://9/>

Tel: (785) 864-3863<tel:%28785%29%20864-3863>
Fax: (785) 864-5335<tel:%28785%29%20864-5335>
Email: ***@***.******@***.***>
ORCID: https://orcid.org/0000-0002-3093-1258
http://ichthyology.biodiversity.ku.edu<http://ichthyology.biodiversity.ku.edu/>
    A  :             A  :             A  :
 }<(((_°>.,.,.,.}<(((_°>.,.,.,.}<)))_°>
    V                V                V

From: Grant Fitzsimmons ***@***.***>
Sent: Monday, March 4, 2024 3:10 PM
To: specify/specify7 ***@***.***>
Cc: Bentley, Andrew Charles ***@***.***>; Mention ***@***.***>
Subject: [specify/specify7] Add native support for MIDS level (Issue #4604)


This proposed framework will integrate to clarify the depth of digitization and the minimum information captured at each level:
https://www.tdwg.org/community/cd/mids/

@acbentley<https://github.com/acbentley> mentioned this to me today and it seems like a great suggestion. Elspeth at RBGE is one of the conveners, so it would be a great opportunity to discuss and collaborate with their team if we choose to pursue implementation.

We may be able to automatically calculate the MIDS level for an individual record. We could add these to the form, statistics page, etc.

Minimal implementation would simply to have a midsLevel field and allow users to enter a value.

4.1 Information elements expected at MIDS level 0

The information elements expected to be present in digital specimen data published at MIDS level 0 are listed in Table 2 below. The terms being developed for the information elements can be found in the TDWG MIDS GitHub site under MIDS-0<https://github.com/tdwg/mids/issues?q=is%3Aissue+is%3Aopen+label%3AMIDS-0>.

Table 2: MIDS Level 0: Expected elements, mappings and recommendations for a bare record
MIDS information element
Definition
Purpose
Required (Bio/Geo/Paelaeo)
1
PhysicalSpecimenID
A unique identity for the specimen within the curating institution. Whatever the institution uses to uniquely identify the item. For example: DOI, stable identifier, catalogue number, barcode, etc.
To allow the curator/researcher to identify the physical specimen the data refer to. To allow citation of the specimen. To enable the attachment of additional information to the specimen record.
Yes (all)
2
Organization
A term to indicate in which institution the specimen is held. This may include an institution code and an institution identifier.
To allow a user to locate the physical location of the specimen.
Yes (all)
Example

Here's an example of what fields would be required to meet each level (each level is assumed to have all the values from the level before it)

Issues arise when you consider users store these values in a number of different fields. Some of the required information doesn't have a specific field to capture it in Specify, meaning users are likely capturing it in a text field. It's hard to make a generalization in those cases and difficult to auto-calculate.

MIDS Level 0

  *   Catalog Number

MIDS Level 1

  *   Organization Name (where it physically belongs)
  *   Determination with a name
  *   Specimen Type
  *   Prep Type
  *   License
  *   Timestamp Modified

MIDS Level 2

  *   Collecting Event, date, collectors, etc.
  *   Geological Age (if geo)
  *   Type
  *   Attachments/media

and so on and so forth.

Here's the working draft on the current repository:
https://github.com/tdwg/mids/blob/working-draft/current-draft%20/MIDS-definition-v0.17-13Jul2023.md#41-information-elements-expected-at-mids-level-0

-

**Comment by g1rly-c0d3r** [comment]
linked in https://github.com/specify/specify7/issues/8213
